### DISCUSSION TITLE
gafana dashboard selector + namespaced dashboards

### DISCUSSION THREAD

**Discussion by zfrhv** [grafana/grafana#45999] [Data Sources]
in kubernetes:
would be cool if grafana had dashboardSelector just as prometheus has ruleSelector.

each dashboard would be created in some namespace, and then it would be possible to manage permissions for dashboard:
each person can manage dashboards only in namespace which he has permissions on. (just by creating/deleting dashboard instance in his own namespace)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
